backend/typescript/services/implementations/EntityServicePg.ts

```diff
@@ -106,18 +154,21 @@ class EntityService implements IEntityService {
       enumField: resultingEntity.enum_field,
       stringArrayField: resultingEntity.string_array_field,
       boolField: resultingEntity.bool_field,
+      fileName,
     };
   }
 
   async deleteEntity(id: string): Promise<void> {
     try {
-      const deletedEntity: number | null = await PgEntity.destroy({
+      const entityToDelete = await PgEntity.findByPk(id, { raw: true });
+      const deleteResult: number | null = await PgEntity.destroy({
         where: { id },
       });
 
-      if (!deletedEntity) {
+      if (!entityToDelete || !deleteResult) {
```

Comment:
Isn't the `!deleteResult` check sufficient?

Reply:
In practice it should be, I just added it in because entityToDelete has type PgEntity | null, so I wanted to remove the TS error in the deleteFile call
Is there a better way to handle this?

Reply:
Ok I see, this works 👍